### PR TITLE
Remove redundant suppression

### PR DESCRIPTION
**Pull request by Philzen**

### CODE REVIEW

**Comment by Philzen** [comment]
As per https://github.com/FasterXML/jackson-annotations/pull/250#issuecomment-2080120796, moved this commit into a separate PR.

File: src/main/java/com/fasterxml/jackson/core/base/ParserBase.java [hunk 1]
```diff
@@ -439,7 +439,6 @@ public boolean hasTextCharacters() {
         return false;
     }
 
-    @SuppressWarnings("resource")
```
Comment (cowtowncoder) [inline src/main/java/com/fasterxml/jackson/core/base/ParserBase.java:442]: I don't see this as redundant on Eclipse.

File: src/main/java/com/fasterxml/jackson/core/Base64Variant.java [hunk 2]
```diff
@@ -669,7 +669,6 @@ public String encode(byte[] input, boolean addQuotes, String linefeed)
      *
      * @throws IllegalArgumentException if input is not valid base64 encoded data
      */
-    @SuppressWarnings("resource")
```
Comment (cowtowncoder) [inline src/main/java/com/fasterxml/jackson/core/Base64Variant.java:672]: Does not exist in 2.18 any longer (converted to try () block).

**Comment by cowtowncoder** [comment]
No longer relevant; one warning removed by 2.18, the other seems legit.

**Pull request closed without merging**